libcob/cconv.c

```diff
@@ -341,21 +380,83 @@ const cob_u8_t cob_ascii_ibmebcdic[256] = {
 	0x3F, 0x3F, 0x3F, 0x3F, 0x3F, 0x3F, 0x3F, 0x3F
 };
 
-enum ebcdic_table cb_ebcdic_table = CB_EBCDIC_DEFAULT;
+const char *
+cob_get_collation_id_as_string (int col_id)
```

Comment:
We may replace this as `cob_get_name_for_collation_id` or drop it completely; the replacement is only reasonable if we expect it to be called later for informational purposes (it currently won't be called in this PR).

Reply:
> Note: the state on this PR - even with the requested changes - is quite good; and while it is a weekend... @ddeclerck @nberth the chances to get it into 3.2 after this weekend are... not good; so to make that available soon you possibly want to have a look at this.

I think this should be  mostly alright. I did what I could considering my current working conditions (I'm in a train at the moment, it's pretty noisy).

Still unsure about the `get_collation_name` function : the codegen needs it to write the actual table name corresponding to the table id that is stored in the flag (or I could just have stored the flag as a string).